daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1050,13 +1050,16 @@ private[lf] object SBuiltin {
 
       if (explicitChoiceAuthority) {
         val authorizers = authorizersWhenExplicit
-        val holding = machine.ptx.context.info.authorizers
+        val signatories = contract.signatories
+        val holding = controllers.union(signatories)
```

Comment:
(Not asking for change)

In such a case I like using the infix notation for operator like `max`, `union`,  ie. ```controllers union signatories```.
Note that scala 3 will for use of backtick à la Haskell, ie. ```controllers `union` signatories```. 
One case use also  `|` operator  which is an alias for `union`.